spec/features/user_show_spec.rb

```diff
@@ -1,6 +1,7 @@
 require 'rails_helper'
 
 RSpec.describe 'User index page', type: :feature do
+  # users show page integration tests
   let!(:user1) { create(:user, name: 'John Doe', photo: 'clement-m-F_-0BxGuVvo-unsplash.jpg') }
 
   before { visit user_path(user1) }
```

Comment:
- [ ] Kindly add tests that assert you can see the user's `username`, `profile pic`, `the number of posts a user has written`, `user's bio`, `the 3 most recent posts for a user`, `Tests that redirects to the user's post's index page when you click on a see all posts`, `Tests that asserts you can see a button that lets me view all of a user's posts`.